packages/regl-worldview/src/stories/assertionStories/GLText.stories.js

```diff
@@ -0,0 +1,145 @@
+// @flow
+
+//  Copyright (c) 2018-present, GM Cruise LLC
+//
+//  This source code is licensed under the Apache License, Version 2.0,
+//  found in the LICENSE file in the root directory of this source tree.
+//  You may not use this file except in compliance with the License.
+
+/* eslint-disable react/display-name */
+
+import { storiesOf } from "@storybook/react";
+import expect from "expect";
+import React from "react";
+import { withScreenshot } from "storybook-chrome-screenshot";
+
+import Cubes from "../../commands/Cubes";
+import GLText from "../../commands/GLText";
+import { clickAtOrigin, WorldviewWrapper } from "../worldviewAssertionUtils";
+import { assertionTest } from "stories/assertionTestUtils";
+
+function textMarkers({
+  text,
+  billboard,
+  background = false,
+}: {
```

Comment:
```suggestion
}: {|
```

We try to use [exact object types](https://flow.org/en/docs/types/objects/#toc-exact-object-types) as much as possible since then Flow will complain if you accidentally pass in unknown fields (useful for spelling mistakes, etc).

Reply:
Great! I updated my tests to use exact objects.